src/handlers.rs

```diff
@@ -79,11 +82,9 @@ pub async fn ingest(
         }
         Err(e) => {
             error!(payload = ?body_string, "failed parsing: {:?}", e);
+            return StatusCode::UNPROCESSABLE_ENTITY.into_response();
         }
     }
-    return Response::builder()
-        .status(StatusCode::OK)
-        .header("x-vercel-verify", state.vercel_verify)
-        .body(Body::empty())
-        .expect("Defined Responses to be infalliable.");
+
+    state.ok_response()
```

Comment:
```suggestion
    return state.ok_response();
```
I'd prefer an explicit return in this case.

Reply:
Done